packages/edit-site/src/components/global-styles/screen-typography.js

```diff
@@ -22,9 +30,7 @@ function ScreenTypography() {
 			/>
 			<div className="edit-site-global-styles-screen-typography">
 				<VStack spacing={ 6 }>
-					{ ! window.__experimentalDisableFontLibrary && (
-						<FontFamilies />
-					) }
+					{ fontLibraryEnabled && <FontFamilies /> }
```

Comment:
If we remove the `window.__experimentalDisableFontLibrary` from the JS, should we also remove it from the PHP?

https://github.com/WordPress/gutenberg/blob/0e48342278e3991f474759e2a537b3ad15d58b37/lib/load.php#L188-L195

Reply:
Yes!

Reply:
done